IPython/core/magics/execution.py

```diff
@@ -19,16 +19,8 @@
 import re
 from pdb import Restart
 
-# cProfile was added in Python2.5
-try:
-    import cProfile as profile
-    import pstats
-except ImportError:
-    # profile isn't bundled by default in Debian for license reasons
-    try:
-        import profile, pstats
-    except ImportError:
-        profile = pstats = None
+import cProfile as profile
+import pstats
```

Comment:
same here `# profile isn't bundled by default in Debian for license reasons` so this code is necessary for debian.

Reply:
I saw that, but I assumed that the cProfile import couldn't fail since it was added in 2.5. Is the cProfile import ever expected to fail?

Reply:
As far as I can tell debian used to build python w/o some modules;  it seem to not be the case anymore. But yes `import profile` used to be able to fail.